integreat_cms/cms/constants/roles.py

```diff
@@ -41,6 +43,10 @@
 #: Choices to use these constants in a database field
 CHOICES = ROLES + STAFF_ROLES
 
+EXTENDED_CHOICES = CHOICES + [
+    (LIMITED_PAGE_MANAGER, _("Limited page manager")),
+]
+
```

Comment:
I did so to avoid errors in the old migration files without changing them.
If the new role is added to the constant ROLES, role update functions in migration files try an update for the new role, which didn't exist at the time and gives an error (I think that was None Type Object).

Is there a way to avoid this or is it okay to change the old migration files?